Makefile

```diff
@@ -2,6 +2,8 @@ USER=jupyter
 REPO=datascience-notebook
 TAG=786611348de1
 .PHONY : html pdf clean notebook
+bash:
+	docker run --rm -v `pwd`/notebooks:/home/jovyan/work -it $(USER)/$(REPO):$(TAG) bash
```

Comment:
looks like your personal $HOME directory is hard-coded here

Reply:
Thanks for the feed back.

No person directory from host is hard-coded, that is why `pwd` is used.

User `/home/jovyan` is an **unprivileged** user defined in the base docker image, see 
[README.md](https://github.com/jupyter/docker-stacks/blob/master/base-notebook/README.md).